### DISCUSSION TITLE
Redirect Page (with Page ID[?])

### DISCUSSION THREAD

**Discussion by nurbaitifitriyani** [mecha-cms/mecha#206] [Q&A] [answered]
Mas,

Saya punya kasus soal _redirect_ sebuah `page`. Misalnya `http://example.com/kanal/youtube` ingin saya alihkan ke `https://youtube.com/username`.

 Biasanya di Blogger saya menggunakan kondisional `page.id`.  Jika di laman dengan `id` sekian, maka alihkan ke `url` ini.

Kalau saya pakai link yang diinputkan di sebuah laman untuk _redirect_ page, jelas semua page yang mempunyai link eksternal akan dialihkan otomatis.

Dalam kasus saya, saya hanya ingin page tertentu yang di-_redirect_.

Apakah Mecha punya kondisional untuk page? Ataukah bisa dilakukan dengan skrip PHP yang fleksibel.

**Comment by taufik-nurrohman** [comment, marked as answer]
Ada kok mba, ekstensinya. Namanya [kick](https://github.com/mecha-cms/x.kick).

**Reply by nurbaitifitriyani** [reply]
Oh kick bisa digunakan untuk eksternal link ya? Saya kira hanya internal saja.

**Reply by taufik-nurrohman** [reply]
Yang internal cuma bagian masukan saja, kalau bagian luarannya bebas diarahkan ke mana saja.
